images/ansible-operator-2.11-preview/Pipfile

```diff
@@ -9,10 +9,7 @@ ansible-runner-http = "==1.0.0"
 ipaddress = "==1.0.23"
 openshift = "~=0.13.0"
 jmespath = "==0.10.0"
-# cryptography needs to be pinned to 3.3.2 as this is the last version
-# before its setup requires rust, which is not available via RPM in the
-# base image. This pin should be re-evaluated once the base image is updated.
-cryptography = "==3.3.2"
+cryptography = "*"
```

Comment:
this can just be `cryptogtaphy`, no = necessary, will default to latest

Reply:
should i change it?

Reply:
If I change it to just `cryptography`, pipenv install fails with error: `pipenv.vendor.toml.decoder.TomlDecodeError: Key name found without value.`.

So I think better to go with current syntax as its key-pair. @asmacdo please suggest.